docs/source/using/image-caching.adoc

```diff
@@ -0,0 +1,37 @@
+include::variables.adoc[]
+
+= Image Caching
+:icons:
+:toc: macro
+:toc-title:
+:toclevels: 2
+
```

Comment:
`toc::[]` is  missing here to display horizontal bar.

![screenshot from 2017-10-24 15-04-23](https://user-images.githubusercontent.com/1132451/31935872-97d17278-b8cd-11e7-8ce1-7f238a0da677.png)

Reply:
let me check ...